bin/nginx.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
We won't need this file and the new environment variable in .env for a couple of reasons:
* For this example, we're going to skip over the Content-Security-Policy for the backend. The `culper` project that we pulled this from doesn't allow the public to hit the API backend. It's just an API for the frontend to use.
* The Content-Security-Policy would need the URL of the API backend to be effective. `self` on line 50 of the NGINX template already refers to the web host.